h2o-py/h2o/explanation/_explain.py

```diff
@@ -1319,6 +1327,8 @@ def ice_plot(
                 )[0]
             )
             encoded_col = tmp.columns[0]
+            if (not is_factor and centered):
```

Comment:
nit/style: Stylistic preference is not to use parenthesis around simple conditional statements

Reply:
[8438350](https://github.com/h2oai/h2o-3/pull/6059/commits/8438350b8fb91a2bddbdc12272173b5877de28b3)